xml/System.Net.Security/SslStream.xml

```diff
@@ -3441,7 +3490,7 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.SslApplicationProtocol</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The TLS Application protocol which is negotiated after this <see cref="T:System.Net.Security.SslStream" /> authenticate.</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>Available once authentication via one of of the AuthenticateAs... methods has been successfully processed.</remarks>
```
Probably worth adding links to `AuthenticateAsClient/Server`.

Also it comes from [ALPN](https://tools.ietf.org/html/rfc7301) which might be worth mentioning, or not.